src/plugins/plugin-run-script.ts

```diff
@@ -0,0 +1,49 @@
+import execa from 'execa';
+import npmRunPath from 'npm-run-path';
+const cwd = process.cwd();
+
+export function runScriptPlugin({cmd, watch: _watchCmd}: {cmd: string; watch?: string}) {
+  const cmdProgram = cmd.split(' ')[0];
+  const watchCmd = _watchCmd && _watchCmd.replace('$1', cmd);
+
+  return {
+    name: `run:${cmdProgram}`,
+    input: [],
+    output: [],
+    async run({isDev, log}) {
+      const workerPromise = execa.command(isDev ? watchCmd || cmd : cmd, {
+        env: npmRunPath.env(),
+        extendEnv: true,
+        shell: true,
+        cwd,
+      });
+      const {stdout, stderr} = workerPromise;
```

Comment:
Should this be `await` since this is a promise?

Reply:
This is an execa thing, `stdout` and `stderr` exist as streams on the promise. Agreed that its a little funky.

TypeScript makes this more obvious, so I don't think we need a comment, but lmk if you'd like one

Reply:
Oh, that's fine with me. I haven't used `execa` before. As long as TS has our back, I'm good with this 👍